pydm/widgets/timeplot_curve_editor.py

```diff
@@ -0,0 +1,71 @@
+from ..PyQt.QtGui import QDialog, QVBoxLayout, QHBoxLayout, QTableView, QAbstractItemView, QSpacerItem, QSizePolicy, QDialogButtonBox, QPushButton, QItemSelection
+from ..PyQt.QtCore import Qt, pyqtSlot
```

Comment:
Luckily, the variable exists.  I'm going to make it so that you can import QItemSelection from PyQt.QtCore OR PyQt.QtGui, so that people used to PyQt4 and PyQt5 both get the behavior the expect.